autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/viewing/AutofillManagementCredentialsMode.kt

```diff
@@ -354,26 +363,47 @@ class AutofillManagementCredentialsMode : DuckDuckGoFragment(R.layout.fragment_a
         invalidateMenu()
     }
 
+    private suspend fun preloadFavicon(credentials: LoginCredentials) {
```

Comment:
when i first read `preload` i was thinking it was pre-emptively trying to fetch the favicon from the web... maybe we could rename this function to `showPlaceholderFavicon` ?